docs/_source/getting_started/installation/server_configuration.md

```diff
@@ -35,7 +35,9 @@ You can set following environment variables to further configure your server and
 
 - `ARGILLA_EXACT_ES_SEARCH_ANALYZER`: Default analyzer for `*.exact` fields in textual information (Default: "whitespace").
 
-- `METADATA_FIELDS_LIMIT`: Max number of fields in the metadata (Default: 50, max: 100).
+- `ARGILLA_METADATA_FIELDS_LIMIT`: Max number of fields in the metadata (Default: 50, max: 100).
+
+- `ARGILLA_METADATA_FIELD_LENGTH`: Max length supported for the string metadata fields. Values containing higher than this will be truncated (Default: 128).
```

Comment:
- `ARGILLA_METADATA_FIELD_LENGTH`: Max length supported for the string metadata fields. Higher values will be truncated. Abusing this may lead to Elastic performance issues (Default: 128).

Reply:
done